src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -224,27 +238,27 @@ public Builder authProvider(AuthProvider authProvider)
             return this;
         }
 
-        public Builder throttle(int throttle)
+        public Builder throttleMegabits(int throttleMegabits)
```

Comment:
I am kind of worried about the similarity of the names... It could be prone to oversights. 
For example, code auto-completion may put `throttleMegabits`, while the actual intent is to have `throttleMebibytes`. 
How about not introducing a new method with the similar name, but adding java docs to indicate the input `throttle` (better rename it to `throttleMegabits`) is in the megabits, and the method is deprecated. 
@ekaterinadimitrova2 , what do you think?

Reply:
Oh so you suggest we remove the throttleMegabits and end up only with the old deprecated throttle and the new throttleMebibytes
That also should work. Honestly, people again can make the same mistake and ignore throttle was in megabits but I guess less methods is better. Also, if we decide to get rid of the megabits version for those properties on a major version.

better rename it to throttleMegabits --> I think any renaming here will be breaking change?

Reply:
> better rename it to throttleMegabits

I was not clear on what is "it". I mean to say that it is better to just rename the _input parameter_ from `throttle` to `throttleMegabits`. Instead, we have 

```java
        @Deprecated 
        public Builder throttle(int throttleMegabits)
        {
           ...
        }
```